app/src/modelProviders/fine-tuned/serializers.ts

```diff
@@ -5,11 +5,13 @@ const FUNCTION_CALL_TAG = "<function>";
 const FUNCTION_ARGS_TAG = "<arguments>";
```

Comment:
This file makes sense but I'd feel more comfortable if we had a couple of tests here with both tool-call and function inputs for sanity.

Reply:
Good idea, I caught a bug while testing.